src/CardContent/Infographic/Infographic.tsx

```diff
@@ -43,10 +50,13 @@ const Infographic = ({ media, version }: InfographicProps) => {
         width: 368
       }}
     >
+      {showSkeleton && <Skeleton width="100%" />}
       <CardMedia
         component="img"
-        image={isVisible ? media : undefined}
-        loading="lazy"
+        alt={hasError ? "card-infographic" : ""}
+        src={isVisible ? media : undefined}
```

Comment:
we incorrectly used the `image` prop until now, which is meant to be used if we are using `component="div"`. This had the side effect of not triggering the `onError` event. Fixed it by switching to src. I also added an `alt` text we missed until now